Dockerfile

```diff
@@ -1,10 +1,15 @@
-FROM renovate/ubuntu:18.04@sha256:f94cb492cb94e913186e477168dff87ab4c12c08a445d65d511c0cc628369d69
+FROM renovate/buildpack@sha256:d1b69a01a9bf536480a1fbfee979f5718edecc91d8e411bcd47bd9f6156844a9
 
-USER root
+ARG JAVA_VERSION=8
 
-RUN apt-get update && apt-get install -y default-jre-headless && \
-    rm -rf /var/lib/apt/lists/*
+LABEL maintainer="Rhys Arkins <rhys@arkins.net>"
+LABEL name="renovate-java"
+LABEL org.opencontainers.image.source="https://github.com/renovatebot/docker-java" \
+      org.opencontainers.image.url="https://renovatebot.com" \
+      org.opencontainers.image.licenses="MIT" \
```

Comment:
Is this correct for openjdk though?

Reply:
🤔 i thought it was for our image, but it's ment for the containing software, so i'll remove licenses